### ISSUE TITLE
Fix coin weight calculation

### ISSUE THREAD

**Issue by orffen** [orffen/basicfantasyrpg#64]
```JavaScript
    // Iterate through money, add to carried weight
    if (context.data.money) {
      let gp = Number(context.data.money.gp.value);
      gp += context.data.money.pp.value * 5;
      gp += context.data.money.ep.value / 5;
      gp += context.data.money.sp.value / 10;
      gp += context.data.money.cp.value / 100;
      carriedWeight._addWeight('*', gp);  // '*' will calculate GP weight
    }
```

Change to:

```javascript
    // Iterate through money, add to carried weight
    if (context.data.money) {
      let gp = Number(context.data.money.gp.value);
      gp += context.data.money.pp.value;
      gp += context.data.money.ep.value;
      gp += context.data.money.sp.value;
      gp += context.data.money.cp.value;
      carriedWeight._addWeight('*', gp);  // '*' will calculate GP weight
    }
```

**Comment by DC23** [comment]
I can do this tomorrow as part of the PR I'll submit.

**Comment by DC23** [comment]
@orffen: GitHub didn't detect that my commit comment of "resolves #64" on 
b1df490 meant that commit fixed this issue. I tried to link this issue to the PR again by tagging it in the PR comment, but GitHub didn't make the connection then either. You'll have to just manually close this issue after merging the PR.

I thought that in this AI age, GitHub would realise that 'resolves' means the same as 'fixes', but no, apparently not.
